src/main/java/com/gengo/client/GengoClient.java

```diff
@@ -51,33 +56,19 @@ public GengoClient(String publicKey, String privateKey)
     public GengoClient(String publicKey, String privateKey, boolean useSandbox)
     {
         super(publicKey, privateKey);
-        setUseSandbox(useSandbox);
-    }
-
-    /**
-     * @return true iff the client is using the sandbox
-     */
-    public boolean getUseSandbox()
-    {
-        return SANDBOX_BASE_URL.equals(baseUrl);
-    }
-
-    /**
-     * Set the client to use the sandbox or the live service.
-     * @param use true iff the client should use the sandbox
-     */
-    public void setUseSandbox(boolean use)
-    {
-        baseUrl = use ? SANDBOX_BASE_URL : STANDARD_BASE_URL;
+        this.flagUseSandbox = useSandbox;
     }
 
     /**
-     * Set a custom base URL. For development testing purposes only.
-     * @param baseUrl a custom API base URL
+     * Select URL which requests are sent for.
```

Comment:
It sounds as if this method accepts a request as a parameter.
I guess "select" is just a implementation detail and the responsibility of this method is to return an appropriate URL.